src/components/universal/core/Badge/Badge.tsx

```diff
@@ -0,0 +1,21 @@
+import { panda } from "generated/panda/jsx";
+import { badge } from "generated/panda/recipes";
+
+import type { BadgeVariantProps } from "generated/panda/recipes";
+import type { ComponentPropsWithoutRef } from "react";
```

Comment:
Is this used because of what Park UI has done (rather than just `ComponentProps`)? I believe they do that because of using React's context API. But I could be wrong! Will test further in a bit when doing manual testing locally! Just figured I would ask.

Reply:
That is what ParkUI does.